src/components/Collapse/Collapse.vue

```diff
@@ -1,21 +1,28 @@
 <template>
-    <div
-        class="d-inline-block"
-        data-bs-toggle="collapse"
-        :data-bs-target="`#${id}`"
+    <slot
+        :id="id"
+        name="toggle"
     >
-        <slot
-            name="toggle"
-        />
-    </div>
+        <div
+            class="collapse-toggle"
+            data-bs-toggle="collapse"
+            :data-bs-target="`#${id}`"
+            v-bind="$attrs"
+        >
+            <slot
+                :id="id"
+                name="toggle-content"
```

Comment:
Should be `kebabCase` to be consistent with Dropdown.vue

```suggestion
                name="toggleContent"
```